custom_components/tuya_local/devices/kogan_lx10_vacuum.yaml

```diff
@@ -0,0 +1,259 @@
+name: Kogan LX10 Robot Vacuum
+products:
+  - id: ldb9ygikft5awhlu
+    name: Kogan LX10 Robot Vacuum
+primary_entity:
+  entity: vacuum
+  dps:
+    - id: 101
+      name: clean_switch
+      type: boolean
+    - id: 102
+      name: pause_switch
+      type: boolean
+    - id: 103
+      name: charge_switch
+      type: boolean
+    - id: 104
+      name: clean_mode
+      type: string
+      mapping:
+        - dps_val: smart
+          value: smart
+        - dps_val: pose
+          value: pose
+        - dps_val: zone
+          value: zone
+        - dps_val: backcharge
+          value: backcharge
+    - id: 105
+      name: robot_state
+      type: string
+      mapping:
+        - dps_val: idle
+          value: idle
+        - dps_val: pointing\areaing
```

Comment:
I'm not sure if backslash is going to get special treatment here, better avoid it for the value, and for dps_val, do some testing - it may need to be doubled to escape it, and/or the string may need double quotes.

Reply:
Without quotes, the backslash appears to be interpreted correctly. When adding double quotes, it would need to be double escaped.